backend/schemas.py

```
@@ -109,3 +109,57 @@ class UserUpdate(BaseModel):
+    method: str  
+    trx_id: str  
+    status: str  
+    created_at: datetime  
+  
+    class Config:  
+        from_attributes = True  
+  
+class OrderCreate(BaseModel):  
+    user_id: int  
+    status: str  
+    order_items: List[OrderItemCreate]  
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)